sentry_sdk/integrations/aws_lambda.py

```diff
@@ -140,25 +135,39 @@ def __init__(self, timeout_warning=False):
     @staticmethod
     def setup_once():
         # type: () -> None
-        import __main__ as lambda_bootstrap  # type: ignore
-
-        pre_37 = True  # Python 3.6 or 2.7
-
-        if not hasattr(lambda_bootstrap, "handle_http_request"):
-            try:
-                import bootstrap as lambda_bootstrap  # type: ignore
 
-                pre_37 = False  # Python 3.7
-            except ImportError:
-                pass
+        # Python 2.7: Everything is in `__main__`.
```

Comment:
This is the crucial bit, where we do not attempt to import but rather see what's imported